tests/test_cli.py

```diff
@@ -1093,3 +1091,44 @@ def test_input_directory(runner, project, run):
         str(p.relative_to(cwd))
         for p in inputs.rglob('*') if p.name != '.gitkeep'
     ) == set(result.output.strip().split('\n'))
+
+
+def test_config_manager_creation(client):
+    """Check creation of configuration file."""
+    path_ = client.config_path
+    assert str(path_).endswith('renku.ini')
```

Comment:
this doesn't test that it was created but that the path contains `renku.ini` - do you want to assert that the file exists?